src/player/features/seek/model/SeekFeature.test.ts

```diff
@@ -7,12 +7,18 @@ import { SeekFeature } from "./SeekFeature";
 describe("SeekFeature", () => {
   test("should seek the song", () => {
     const currentTime = 10;
+    const currentVolume = 0.5;
     const songDuration = 260;
     const seekPosition = 33;
     const expectedTime = 85.8;
 
     const player = new Player({
-      audioManager: new InMemoryAudioManager(currentTime, songDuration, false),
+      audioManager: new InMemoryAudioManager(
+        currentTime,
+        currentVolume,
+        songDuration,
+        false,
+      ),
```

Comment:
Ah you can see there this setup is not ideal ^^

You're working on a `volume` feature but we have to make a modification in this use case. We'll improve that later 👍

Reply:
Agree, add a TODO.